### DISCUSSION TITLE
Looking for a set of v5 compatible packages for use with RN 0.65/0.66

### DISCUSSION THREAD

**Discussion by JeremyBradshaw7** [react-navigation/react-navigation#10574] [Q&A]
I've been using React-Navigation v5 for a while now. But everything seemed to go haywire after trying to upgrade from RN 0.64.1 through to 0.66.4. Under RN 0.64.1 I had these v5 navigation-related package versions, which all worked nicely together:

```
		"@react-native-community/masked-view": "^0.1.11",
		"@react-navigation/bottom-tabs": "^5.11.10",
		"@react-navigation/drawer": "^5.12.5",
		"@react-navigation/native": "^5.9.4",
		"@react-navigation/stack": "^5.14.5",
		"react-native-gesture-handler": "^1.10.3",
		"react-native-reanimated": "^2.2.0",
		"react-native-safe-area-context": "^3.3.2",
		"react-native-screens": "^3.1.1",
```

I didn't want to upgrade these packages unless I had to, but in upgrading to RN0.65 I found I had to upgrade `react-native-reanimated` to `2.2.4` due to [this](https://github.com/software-mansion/react-native-reanimated/issues/3048) issue, and also configure the babel plugin.

On upgrading to RN 0.66.4 I further found I had to upgrade `react-native-reanimated` to `2.8.0`, although I have lost the reason why. This all works for standard builds and for debug builds UNTIL I turn on "Debug to Chrome", whereupon my main `Drawer.Navigator` component does not render at all, it just shows a white screen, and should I bypass that then no navigations are performed, & the screen then seems to hang, I suspect because of the reanimated package as that's the only navigation-related package I have changed.

Does anyone have a set of v5 package versions that works reliably in combination with RN 0.65 and/or 0.66 ? Or should I take the leap and try to upgrade all the navigation packages to v6 ?

**Comment by raajnadar** [comment]
If you use expo I would suggest to check the module compatibility with `expo install <package-name>` cmd.

Yes It is highly recommended to upgrade to v6, there is very less friction between v5 & v6 so it should be smooth.
https://reactnavigation.org/docs/upgrading-from-5.x

Once you are with react navigation v6, you can use the latest version of other dependant libraries without any issue, ake sure after your upgrade process is done you clear the metro cache, `expo start -c` or `npx react-native start --reset-cache`

**Comment by JeremyBradshaw7** [comment]
Thanks I may try the v6 upgrade then, I'm not using expo no.
